Ingest updates for split (#198)
Azure/azure-kusto-go: CHANGELOG.md

* [BREAKING] [MAJOR] Split the main module into two packages:
    * azkustodata - contains querying, management APIs.
    * azkustoingest - contains ingestion in all its forms.
* [BREAKING] [MAJOR] Constructing ingest clients is now done using a KustoConnectionStringBuilder, and not a client struct.

> * In addition, passing a default database and table for ingestion is not necessary anymore, and can be done using Options.

Reviewer: ```suggestion
* In addition, passing a default database and table at ingest time is not mandatory anymore, and can be done using Options.
```

?

Author: No, it's in the ctor
